news/2024/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title: "The Followpoint: Hivie, the Man of Anything and Everything"
+date: 2024-11-25 17:00:00 +0000
+series: The Followpoint
+---
+
+Ever heard of the person who does literally anything you can think of? No? Well, in this The Followpoint anniversary interview you'll learn more about such a person!
+
+![](/wiki/shared/news/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything/banner.jpg)
+
+Welcome or welcome back to The Followpoint, a series where we interview various players and community figures to get to know them better! The series' first anniversary happened just two weeks ago, which means we're interviewing someone special both to us and the osu! community in general, [Hivie](https://osu.ppy.sh/users/14102976)!
+
+Hivie is an [osu! team](/wiki/People/osu!_team) member, but is also a [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) leader, [Project Loved](/wiki/Community/Project_Loved) manager, [medal](/wiki/Medals) coordinator, among so many other things. If you've been active in mapping or development discussion over the past years, you've surely seen him around! Funnily enough, [our first interview](https://osu.ppy.sh/home/news/2023-11-10-the-followpoint-ephemeral-the-man-behind-the-medals) was with [Ephemeral](https://osu.ppy.sh/users/102335), who also was a community maintainer and medal developer. That, and the fact that Hivie often helps us with polishing our interviews, makes interviewing him that much more fitting for today!
+
+This interview will look slightly different from the ones you got used to, as we'd like to make it in the spirit of our first one. That means it will be less flow-based, but the questions will still be split into categories.  
+
+I'd also like to say something at the end of this interview, but first things first, let's see what Hivie has to say! :3
```

Comment:
Could probably cut this line, or at least remove the smiley. Adding the outro is fine but I don't think you need to mention it here.

Reply:
the line itself makes sense to keep since it eases the reader into the outro's more personal tone shift, so i've just removed the smiley